src/test/scala/org/broadinstitute/dsde/workbench/sam/RetryableAnyFlatSpec.scala

```diff
@@ -0,0 +1,22 @@
+package org.broadinstitute.dsde.workbench.sam
+
+import org.scalatest.{Canceled, Failed, Outcome, Retries}
+import org.scalatest.flatspec.AnyFlatSpec
+
+trait RetryableAnyFlatSpec extends AnyFlatSpec with Retries {
+
+  val retries = 3
+
+  override def withFixture(test: NoArgTest): Outcome =
+    withFixture(test, 1)
+
+  def withFixture(test: NoArgTest, count: Int): Outcome = {
+    val outcome = super.withFixture(test)
+    outcome match {
+      case Failed(_) | Canceled(_) =>
+        println(s"Test '${test.name}' failed on try $count.")
+        if (count >= retries) super.withFixture(test) else withFixture(test, count + 1)
```

Comment:
I am reading this that after we hit max retries, this will run the test 1 more time?  Is that right?  

Would the alternative be:
```suggestion
        if (count >= retries) outcome else withFixture(test, count + 1)
```
I don't care either way, other than I think this suggestion is ever so slightly more intuitive.  "If we've retried max times, return the last outcome and exit" vs. "If we've retried max times, give it ONE more try just in case."

Reply:
This will still only run the test 3 times. If it fails on `1`st try, and fails on `2`nd try, then `super.withFixture` will be the `3`rd and final try, which is returned.